packages/aws-cdk-lib/cx-api/FEATURE_FLAGS.md

```diff
@@ -1439,22 +1441,6 @@ If the flag is set to false then a custom resource will be created when using `U
 | (not in v1) |  |  |
 | 2.174.0 | `false` | `true` |
 
-### @aws-cdk/aws-route53-targets:userPoolDomainNameMethodWithoutCustomResource
```

Comment:
I believe this section got removed by mistake while resolving merge conflict

Reply:
This is correctly removed as there're two occurrences of the same statement. I removed one of them to make it right.